core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -236,8 +236,11 @@ class Simulator {
    * currently in flight.
    */
   _updateNetworkCapacity() {
+    const inFlight = this._numberInProgress(BaseNode.TYPES.NETWORK);
+    if (inFlight === 0) return;
+
     for (const connection of this._connectionPool.connectionsInUse()) {
-      connection.setThroughput(this._throughput / this._nodes[NodeState.InProgress].size);
```

Comment:
for the historical record: the reason it did this was because the original prototype graph _only_ had network nodes, not cpu ones. @paulirish [noted at the time](https://github.com/GoogleChrome/lighthouse/pull/3187#discussion_r136456496) that we should make sure not to keep including all active nodes when cpu nodes were added to the graph, but we did not remember.